### ISSUE TITLE
APTrust profile with BagIt-Version 1.0 generates mismatched manifest algorithms.

### ISSUE THREAD

**Issue by whikloj** [APTrust/dart#223]
I haven't tried this with the 2.0.1 version [mentioned here](https://github.com/APTrust/dart/issues/221#issuecomment-592033187) but when playing with DART I noticed that if I use the APTrust BagIt profile and switch the `BagIt-Version` to `1.0` I get this bag.

```bash
[/Users/whikloj/.dart/bags] 
> ll An\ Example\ Bag
total 40
-rw-r--r--  1 whikloj  staff   77 26 Feb 14:58 aptrust-info.txt
-rw-r--r--  1 whikloj  staff  350 26 Feb 14:58 bag-info.txt
-rw-r--r--  1 whikloj  staff   54 26 Feb 14:58 bagit.txt
drwxr-xr-x  3 whikloj  staff   96 26 Feb 14:58 data
-rw-r--r--  1 whikloj  staff   46 26 Feb 14:58 manifest-md5.txt
-rw-r--r--  1 whikloj  staff  317 26 Feb 14:58 tagmanifest-sha256.txt
```

I think the inclusion of `manifest-md5.txt` as the default seems like an error, especially due to the inclusion of `tagmanifest-sha256.txt`.

I was trying to fix this as a lesson for myself in Node and DART, but it could take me a while. So thought I should bring it to your attention.

**Comment by diamondap** [comment]
Thanks @whikloj. I actually know what's causing that. I'll work on a fix in the next few days.

**Comment by diamondap** [comment]
This should be fixed in version 2.0.2. Download links are [here](https://github.com/APTrust/dart/blob/master/README.md#installation).

**Comment by diamondap** [comment]
Fixed in v2.0.2 and later.
